### PR TITLE
Scavengers: Viewhunt/Queue improvements/fixes

### PR DESCRIPTION
**Pull request by sparkychildcharlie**
- Add "unrated" hunts to queue system
- fix /forcestarthunt, an add an unrated option for this.
- allow /viewhunt to show the current question while leaving the unknown answer blank

I plan to soon do typescripting with scav stuff so it's like the other chat-plugins

<sub>I swear I am a good neko!!!</sub>

### CODE REVIEW

**Comment by Zarel** [comment]
As far as I know, we have no devs on Scavengers? Someone will need to run these changes past Scavs auth.

**Comment by sparkychildcharlie** [comment]
If i held back on my salty shitposting I would be lol. Anyways regarding this, it has full approval from Meicoo (screenshots on Discord)

File: server/chat-plugins/scavengers.js [hunk 1]
```diff
@@ -1543,7 +1556,7 @@ exports.commands = {
 			"- /scav resetladder - resets the monthly scavenger leaderboard. (Requires: # & ~)",
 			"- /scav setpoints [1st place], [2nd place], [3rd place], [4th place], [5th place], ... - sets the point values for the wins. Use `/scav setpoints` to view what the current point values are. (Requires: # & ~)",
 			"- /scav setblitz [value] ... - sets the blitz award to `value`. Use `/scav setblitz` to view what the current blitz value is. (Requires: # & ~)",
-			"- /scav queue <em>[host] | [hint] | [answer] | [hint] | [answer] | [hint] | [answer] | ...</em> - queues a scavenger hunt to be started after the current hunt is finished. (Requires: % @ * # & ~)",
+			"- /scav queue(unrated) <em>[host] | [hint] | [answer] | [hint] | [answer] | [hint] | [answer] | ...</em> - queues a scavenger hunt to be started after the current hunt is finished. (Requires: % @ * # & ~)",
```
Comment (Zarel) [inline server/chat-plugins/scavengers.js]: I don't think we use the `(unrated)` syntax, it's pretty confusing here. I'd make them different commands in the list.

Comment (sparkychildcharlie) [reply server/chat-plugins/scavengers.js]: What would you prefer instead? I'm just using the same syntax as the /start(mini/official/unratrd/practice)hunt help entry above)

Comment (Zarel) [reply server/chat-plugins/scavengers.js]: > What would you prefer instead?

> I'd make them different commands in the list.

Comment (Zarel) [reply server/chat-plugins/scavengers.js]: Oh, I didn't know they did that for the other ones.

Comment (Zarel) [reply server/chat-plugins/scavengers.js]: I'd at least suggest `queue(rated/unrated)` to make it clearer what's going on there.

**Comment by sparkychildcharlie** [comment]
@Zarel I've updated it :3

**Comment by xfix** [comment]
I'm not a fan of this huge line 183, but the change seems fine.

**Pull request merged**